event-tracking/event-tracking-service/src/main/java/uk/gov/justice/services/subscription/ProcessedEventTrackingRepository.java

```diff
@@ -40,6 +41,15 @@ public class ProcessedEventTrackingRepository {
                     "AND component = ? " +
                     "ORDER BY event_number DESC";
 
+    private static final String SELECT_LESS_THAN_EVENT_NUMBER_IN_DESCENDING_ORDER_SQL =
+            "SELECT event_id, event_number, previous_event_number " +
+                    "FROM processed_event " +
+                    "WHERE source = ? " +
+                    "AND component = ? " +
+                    "AND event_number < ? " +
+                    "ORDER BY event_number DESC " +
```

Comment:
is this order by going to kill us in a 37million or eventually 200million row table?